src/exercise/06.md

```diff
@@ -88,9 +88,11 @@ make this work:
 2. Use the value of the input to determine whether there's an error. There's an
    error if the user typed any upper-case characters. You can check this really
    easily via `const isValid = value === value.toLowerCase()`
-3. if there's an error, set the error state to `'Username must be lower case'`.
+3. If there's an error, set the error state to `'Username must be lower case'`.
    (💰 here's how you do that:
    `setError(isValid ? null : 'Username must be lower case')`)
+4. Finally, display the error in an element with a specific `role=` attribute.
+   (💰 check the test file for a hint.)
```

Comment:
In the extra credit, we're intentionally ambiguous because I want people to do it differently and then wonder why their solution and mine differed. This will help them remember better (even though it might be a little frustrating at first). I don't mind frustrating people a tiny bit if it means they'll remember/understand better :)

```suggestion
4. Finally, display the error in an element.
```

Thanks!